src/Orchard/UI/Resources/MetaEntry.cs

```diff
@@ -5,6 +5,35 @@
 namespace Orchard.UI.Resources {
     public class MetaEntry {
         private readonly TagBuilder _builder = new TagBuilder("meta");
+        
+        public MetaEntry()
+        {
+
+        }
+
+        public MetaEntry(string name = null, string content = null, string httpEquiv = null, string charset = null)
+        {

```

Comment:
that was to meet the requirement of the calling methods and the code segment to keep arguments optional.